lib/mongo/collection/view/writable.rb

```diff
@@ -182,6 +191,7 @@ def delete_many(opts = {})
                   :db_name => collection.database.name,
                   :coll_name => collection.name,
                   :write_concern => write_concern,
+                  :bypass_document_validation => !!opts[:bypass_document_validation], 
```

Comment:
I think this is fine for now as long but we may need to change this later to not send the false value if this fails any tests.

On which note, I don't see tests being run for this PR, so perhaps there are actually some failures due to the false value being passed.

Reply:
The CRUD spec tests are pretty sparse in terms of which options they test, so I wouldn't be shocked if they don't really test bypassDocumentValidation.

Reply:
So agreed that this might need to be cleaned up one day, but it doesn't seem to be failing any tests right now.

Reply:
https://jira.mongodb.org/browse/DRIVERS-1316 filed to address this.